trueline.sh

```diff
@@ -551,6 +553,7 @@ declare -A TRUELINE_SYMBOLS_DEFAULT=(
     [exit_status]=' '
     [git_ahead]=''
     [git_behind]=''
+    [git_codeberg]=''
```

Comment:
Place this before git_github icon (i.e the list is alphabetically ordered)

Reply:
Done.